backend/prompt_studio/prompt_studio_output_manager_v2/output_manager_util.py

```diff
@@ -7,7 +7,10 @@
 class OutputManagerUtils:
     @staticmethod
     def get_coverage(
-        tool_id: str, profile_manager_id: str, prompt_id: str = None
+        tool_id: str,
+        profile_manager_id: str,
+        prompt_id: str = None,
+        is_single_pass: bool = False,
```

Comment:
@jagadeeswaran-zipstack single pass is a cloud feature, why does it need to be referenced or added here? Please add a relevant TODO in the code to remove this from OSS